hack/lint-paths.txt

```diff
@@ -6,6 +6,7 @@ pkg/network/domainspec
 pkg/network/namescheme
 pkg/network/pod/annotations
 pkg/network/vmicontroller
+pkg/preference
```

Comment:
I am worried that these `preferences` won't be understood well. Perhaps, to provide context, we should place `preferences` under `pkg/instancetypes/preferences`?

Reply:
I appreciate the APIGroup is `instancetype.kubevirt.io` but I still think it should be pretty obvious from the owners file alone what the functionality provides. We can even drop some docs in there to make it explicit.

Nesting preferences under `pkg/instancetype/preference` while still having the instancetype logic under `pkg/instancetype` just doesn't sit well in my head tbh.

Reply:
There must be some context around the `preference` directory. 
If you think that nesting is not a good idea then maybe we should name it differently, could be instancetypepreference? 
I'm not sure, but `pkg/preference` doesn't indicate that this is something that belongs to the instancetypes.

Reply:
Okay moved under `pkg/instancetype`.